src/Resources/config/validation/redirection.yaml

```diff
@@ -0,0 +1,11 @@
+Setono\SyliusRedirectPlugin\Model\Redirect:
```

Comment:
The name of this file should be `redirect.yaml`

Reply:
ow indeed